client/my-sites/backup/rewind-flow/restore.tsx

```diff
@@ -334,8 +328,59 @@ const BackupRestoreFlow: FunctionComponent< Props > = ( {
 					}
 				) }
 			</p>
-		</Error>
-	);
+		);
+	};
+
+	const ErrorDetailsAddCredentials = () => {
+		return (
+			<>
+				<p className="rewind-flow__info">
+					{ translate(
+						'An error occurred while restoring your site. You may need to {{linkCredentials}}add your server credentials{{/linkCredentials}}. You can follow the steps in {{linkGuide}}our guide{{/linkGuide}} to add SSH, SFTP, or FTP credentials, and then try to restore again.',
+						{
+							components: {
+								linkCredentials: (
+									<a
+										href={
+											rewindState.canAutoconfigure
+												? `/start/rewind-auto-config/?blogid=${ siteId }&siteSlug=${ siteSlug }`
+												: `${ settingsPath( siteSlug ) }#credentials`
+										}
+									/>
+								),
+								linkGuide: (
+									<a
+										href="https://jetpack.com/support/adding-credentials-to-jetpack/"
```

Comment:
Could we use an `ExternalLink` here?
Also, I think we could benefit on adding Tracks event for these clicks.

Reply:
Sure. Added ExternalLink.

I'll work on adding track events later on Monday

Reply:
Could we try using ExternalLink from `@wordpress/components` instead of `calypso/components/external-link`?
The `target="_blank"` [is already part of it](https://github.com/WordPress/gutenberg/blob/f50e045c1ce75acd026ab760891cd5bf57234fd3/packages/components/src/external-link/index.tsx#L62).

The `calypso/components/external-link` seems to be missing the external link icon that should be at the end of the link.

![CleanShot 2024-07-29 at 15 52 23@2x](https://github.com/user-attachments/assets/e3668013-b945-477b-8208-2ddcce2cbee4)

Should look similar to this:

![CleanShot 2024-07-29 at 15 56 09@2x](https://github.com/user-attachments/assets/ea0e0b7f-fa85-428b-bbbd-8af321492c38)